lib/uploader.ts

```diff
@@ -41,7 +38,10 @@ export class Uploader {
 
 	// Global upload queue
 	private _uploadQueue: Array<Upload> = []
-	private _jobQueue: PQueue = new PQueue({ concurrency: MAX_CONCURRENCY })
+	private _jobQueue: PQueue = new PQueue({
+		// Maximum number of concurrent uploads
+		concurrency: window.OC?.appConfig?.files?.max_parallel_count ?? 5,
```

Comment:
I would directly use `@nextcloud/capabilities` here to get the value and not rely on magic window variables :)

Reply:
Sure, I thought this might be easier for now. Also see https://github.com/nextcloud/server/issues/48462 as I want to get rid of this thing entirely at some point.